sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/HiveSqlParser.scala

```diff
@@ -370,7 +373,8 @@ class HiveSqlAstBuilder extends SparkSqlAstBuilder {
       ctx: TableFileFormatContext): CatalogStorageFormat = withOrigin(ctx) {
     import ctx._
     if (inDriver != null || outDriver != null) {
-      logWarning("INPUTDRIVER ... OUTPUTDRIVER ... clauses are ignored.")
```

Comment:
https://issues.apache.org/jira/browse/HIVE-1546 I checked the discussion. Based on what Carl Steinbach and Ashutosh Chauhan said, 
> Hive doesnt care about INPUTDRIVER and OUTPUTDRIVER

I also searched the code base of hive master. They are not used. Not sure if we should completely delete them? or keep issuing the exceptions. Thanks!